types/xrm/index.d.ts

```diff
@@ -1150,41 +1150,31 @@ declare namespace Xrm {
          * Gets an attribute matching attributeName.
          * @param T An Attribute type.
          * @param attributeNameOrIndex Name of the attribute.
-         * @returns The attribute.
-         */
-        getAttribute<T extends Attributes.Attribute>(attributeNameOrIndex: string | number): T | null;
-
-        /**
-         * Gets an attribute by name or index.
-         * @param attributeNameOrIndex Name of the attribute or the attribute index.
          * @returns The attribute or null if attribute does not exist.
          */
-        getAttribute(attributeNameOrIndex: string | number): Attributes.Attribute | null;
+        getAttribute<T = Attributes.SpecificAttributeTypes>(
```

Comment:
Did you mean `extends` here (and elsewhere)? `=` would imply _anything_ could be returned, but the old code restricted to just `Attributes.Attribute`.

Reply:
Yeah, it should be constrained, I'll push a fix!